core/src/lib/bool_string.cc

```diff
@@ -0,0 +1,26 @@
+/*
+   BAREOS® - Backup Archiving REcovery Open Sourced
+
+   Copyright (C) 2020-2020 Bareos GmbH & Co. KG
+
+   This program is Free Software; you can redistribute it and/or
+   modify it under the terms of version three of the GNU Affero General Public
+   License as published by the Free Software Foundation and included
+   in the file LICENSE.
+
+   This program is distributed in the hope that it will be useful, but
+   WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+   Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+   02110-1301, USA.
+*/
+
+#include "lib/bool_string.h"
+#include <set>
+
+const std::set<std::string> BoolString::true_values{"on", "true", "yes", "1"};
+const std::set<std::string> BoolString::false_values{"off", "false", "no", "0"};
```

Comment:
The initialization order of global variables over multiple translation units is unspecified.
So if someone uses `BoolString` to initialize a global variable, it may use `true_values` and `false_values` before they are initialized.